modules/nf-core/ascat/main.nf

```diff
@@ -157,43 +158,43 @@ process ASCAT {
 
     write.table(QC, file=paste0("$prefix", ".metrics.txt"), sep="\t", quote=F, row.names=F)
 
-    # version export
+    # Version export
     f <- file("versions.yml","w")
     alleleCounter_version = system(paste("alleleCounter --version"), intern = T)
-    ascat_version = sessionInfo()\$otherPkgs\$ASCAT\$Version
+    ascat_version = as.character(packageVersion('ASCAT'))
     writeLines(paste0('"', "$task.process", '"', ":"), f)
-    writeLines(paste("    alleleCounter:", alleleCounter_version), f)
```

Comment:
We got a bunch of new tools in the environment yml. Shouldn't they be reported here?